.gitignore

```diff
@@ -29,6 +29,7 @@ site/e2e/test-results/*
 site/e2e/states/*.json
 site/e2e/.auth.json
 site/playwright-report/*
+site/static/icons.json
```

Comment:
Why are we adding it ti `.gitignore`?

Reply:
I guess it's just habit to ignore generated files